src/importexport/mei/tests/mei_tests.cpp

```diff
@@ -68,11 +75,177 @@ void Mei_Tests::meiReadTest(const char* file, const char* ext)
         return;
     }
 
+    // Compare with the reference MuseScore file
     EXPECT_TRUE(ScoreComp::saveCompareScore(score, fileName + u".mscx", MEI_DIR + fileName + u".mscx"));
+
+    // Save the .mei file for round trip testing
+    bool output = ScoreRW::saveScore(score,  fileName + u".test.mei", exportFunc);
+    EXPECT_TRUE(output);
     delete score;
+
+    // Compare the mei files
+    EXPECT_TRUE(ScoreComp::compareFiles(fileName + u".test.mei", ScoreRW::rootPath() + u"/" + MEI_DIR + fileName + u".mei"));
+
+    /*
+    // Reload the generated .mei file
+    MasterScore* score2 = ScoreRW::readScore(fileName + u".test.mei", true, importFunc);
+    EXPECT_TRUE(score2);
+
+    // Compare with the reference MuseScore file
+    EXPECT_TRUE(ScoreComp::saveCompareScore(score2, fileName + u".mscx", MEI_DIR + fileName + u".mscx"));
+    delete score2;
+    */
```

Comment:
I think we can remove this comment. Since the file at `fileName + u".test.mei"` is supposed to be the equal to the original `.mei` file, `score2` will be exactly the same as `score`, so it does not need to be compared to the mscx file again.